source/features/user-local-time.tsx

```diff
@@ -34,15 +34,15 @@ async function loadCommitPatch(commitUrl: string): Promise<string> {
 }
 
 const getLastCommitDate = cache.function(async (login: string): Promise<string | false> => {
-	for await (const page of api.v3paginated(`users/${login}/events`)) {
+	for await (const page of api.v3paginated(`/users/${login}/events`)) {
 		for (const event of page as any) {
 			if (event.type !== 'PushEvent') {
 				continue;
 			}
 
 			// Start from the latest commit, which is the last one in the list
 			for (const commit of event.payload.commits.reverse() as Commit[]) {
-				const response = await api.v3(commit.url, {ignoreHTTPStatus: true});
+				const response = await api.v3('/' + commit.url, {ignoreHTTPStatus: true});
```

Comment:
I'm not entirely sure, but I'd think that `commit.url` has the WHOLE URL already, including `https:`. Please verify

Reply:
It does and with the current logic the query end up being  repos/username + the link. Maybe we should rather add another option of `absoluteURL`.

Reply:
@fregante mind to give this a quick look?

Reply:
Ah true. You can just detect wether it starts with http and skip any transforms. No options necessary